chapters/en/slides/module3/module3_03.Rmd

```diff
@@ -0,0 +1,245 @@
+---
+params:
+  dynamictitle: "module3_03"
+title: "`r params$dynamictitle`"
+output:
+  md_document:
+    variant: gfm
+    pandoc_args: "--markdown-headings=atx"
+---
+
+```{r setup, include=FALSE}
+source('../../../../slide-setup.R')
+# Remember to also manually update the YAML title above
+```
+
+type: slides
+
+# Comparing many distributions
+
+Notes:
+We have seen have we can use facetting for histograms
+and both faceting and coloring for density plots
+to compare multiple distributions against each other.
+So far we only studied these plots with up to three different distributions to compare,
+but what if we had many more?
```

Comment:
Let's give an idea of how many more - say 20? 50? 100? That will let them imagine how poorly the viz's we have shown this far will work.